.github/workflows/unit-tests.yaml

```
@@ -55,79 +59,42 @@ jobs:
 
     - name: Get deps
       run: |
-          cd ${{ github.workspace }}/src/github.com/snapcore/snapd/ && ./get-deps.sh
+          ./get-deps.sh
 
     - name: Build C
       run: |
-          cd ${{ github.workspace }}/src/github.com/snapcore/snapd/cmd/
+          cd cmd/
           ./autogen.sh
           make -j$(nproc)
 
     - name: Build Go
       run: |
-          go build github.com/snapcore/snapd/...
+          go build ./...
 
     - name: Test C
       run: |
-          cd ${{ github.workspace }}/src/github.com/snapcore/snapd/cmd/ && make distcheck
-
-    - name: Reset code coverage data
-      run: |
-          rm -rf ${{ github.workspace }}/.coverage/
-          COVERAGE_OUT="${{ github.workspace }}/coverage/coverage-${{ inputs.unit-scenario}}.cov"
-          echo "COVERAGE_OUT=$COVERAGE_OUT" >> $GITHUB_ENV
-
-    - name: Test Go
-      if: ${{ inputs.unit-scenario == 'normal' }}
-      run: |
-        cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
-        ./run-checks --unit
+          cd cmd/ && make distcheck
+    
+    - name: Set Snapd debug
+      if: ${{ inputs.snapd-debug }}
+      run: echo "SNAPD_DEBUG=1" >> $GITHUB_ENV
 
-    - name: Test Go (SNAPD_DEBUG=1)
-      if: ${{ inputs.unit-scenario == 'snapd_debug' }}
-      run: |
-        cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
-        SKIP_DIRTY_CHECK=1 SNAPD_DEBUG=1 ./run-checks --unit
+    - name: Set Go test Race
+      if: ${{ inputs.go-test-race }}
+      run: echo "GO_TEST_RACE=1" >> $GITHUB_ENV
 
-    - name: Test Go (withbootassetstesting)
-      if: ${{ inputs.unit-scenario == 'withbootassetstesting' }}
-      run: |
-        cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
-        SKIP_DIRTY_CHECK=1 GO_BUILD_TAGS=withbootassetstesting ./run-checks --unit
+    - name: Set Skip Coverage
```

```suggestion
    - name: Set SKIP_COVERAGE=1
```

Same comment as above. Separately why is this not prefixed with `GO_`? Is that our own variable or something we interact with but external.

---

I believe it's only our own variable. It's used inside `run-checks` to determine whether it should add flags for coverage when running unit tests. If it makes more sense to change it to `GO_SKIP_COVERAGE`, I can definitely do that. I'll just have to let everyone know when that happens so I don't mess people up who run `run-checks` locally.